cmd/copy.go

```diff
@@ -1284,7 +1281,7 @@ func init() {
 	cpCmd.PersistentFlags().BoolVar(&raw.background, "background-op", false, "true if user has to perform the operations as a background operation.")
 	cpCmd.PersistentFlags().StringVar(&raw.acl, "acl", "", "Access conditions to be used when uploading/downloading from Azure Storage.")
 
-	cpCmd.PersistentFlags().BoolVar(&raw.s2sCheckLength, "s2s-check-length", true, "Check the length of a file transferred S2S after the transfer. If there is a mismatch, fail the transfer.")
+	cpCmd.PersistentFlags().BoolVar(&raw.CheckLength, "check-length", true, "Check the length of a file transferred S2S after the transfer. If there is a mismatch, fail the transfer.")
```

Comment:
Please check entire text here. Some of it still refers, incorrectly now, to S2S.

Reply:
perhaps consider making a note that there's an IO cost

---
In reply to: [313170980](https://github.com/Azure/azure-storage-azcopy/pull/548#discussion_r313170980) [](ancestors = 313170980)